src/common/personalizedDigest.ts

```diff
@@ -122,19 +123,60 @@ const getPostsTemplateData = ({
   });
 };
 
+type DynamicAd = {
+  type: string;
+  title: string;
+  link: string;
+  image: string;
+  company_name: string;
+  company_logo: string;
+  cta: string;
+};
+const getEmailAd = async ({
+  user,
+  feature,
+}: {
+  user: User;
+  feature: PersonalizedDigestFeatureConfig;
+}): Promise<DynamicAd | null> => {
+  // TODO: Temporary hardcode 75 check
+  if (
+    isPlusMember(user.subscriptionFlags?.cycle) ||
+    feature.templateId != '75'
+  ) {
+    return null;
+  }
+
+  const ad = await skadiClient.getAd('default_digest', {
+    USERID: user.id,
+  });
+
+  return {
+    type: ad.type,
+    title: ad.value.digest.title,
+    link: ad.value.digest.link,
```

Comment:
Is `link` correct? The fallback ad uses `post_link` there

Reply:
They different types of ads, this one is new, so new naming :P